package.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.0.7",
   "description": "A Geo-toolkit built in Javascript.",
   "main": "terraformer.js",
-  "typings": "index.d.ts",
+  "typings": "terraformer.d.ts",
   "files": [
-    "index.d.ts"
+    "terraformer.d.ts"
```

Comment:
this is nitpicky, but i think its sufficient to reference `terraformer.d.ts`. lets remove pkg.files entirely.

also, lets bite the bullet and point travis at `"node"` instead of `4.x` (as per my comments [here](https://github.com/Esri/terraformer-wkt-parser/pull/29/files#r108294429))